### PR TITLE
DOCAPI-3825: system.metrics, system.events, system.asyncronouse_metrics docs

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation

### CODE REVIEW

File: docs/en/operations/system_tables.md [hunk 1]
```diff
@@ -8,9 +8,39 @@ They are located in the 'system' database.
 
 ## system.asynchronous_metrics {#system_tables-asynchronous_metrics}
 
-Contain metrics used for profiling and monitoring.
-They usually reflect the number of events currently in the system, or the total resources consumed by the system.
-Example: The number of SELECT queries currently running; the amount of memory in use.`system.asynchronous_metrics`and`system.metrics` differ in their sets of metrics and how they are calculated.
+Contains metrics which are calculated periodically. For example, RAM in use.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: * I'd add "periodically **in background**"
* **the amount of** RAM in use

File: docs/en/operations/system_tables.md [hunk 2]
```diff
@@ -8,9 +8,39 @@ They are located in the 'system' database.
...
+│ UncompressedCacheCells                  │          0 │
+│ MarkCacheFiles                          │          0 │
+└─────────────────────────────────────────┴────────────┘
+```
+
+**See Also**
+
+- [system.metrics](#system_tables-metrics) — Contains instantly calculated metrics.
+- [system.events](#system_tables-events) — Contains a number of happened events.
+- [Monitoring](monitoring.md) — Base concepts of ClickHouse monitoring.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: maybe put this first in a list?

File: docs/en/operations/system_tables.md [hunk 3]
```diff
@@ -89,9 +119,35 @@ Note that the amount of memory used by the dictionary is not proportional to the
...
 
-Contains information about the number of events that have occurred in the system. This is used for profiling and monitoring purposes.
-Example: The number of processed SELECT queries.
-Columns: 'event String' – the event name, and 'value UInt64' – the quantity.
+Contains information about the number of events that have occurred in the system. For example, in the table, you can find how many `SELECT` queries are processed from the moment of ClickHouse server start.
+
+Columns:
+
+- `event` ([String](../data_types/string.md)) — Event name.
+- `value` ([UInt64](../data_types/int_uint.md)) — Quantity.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: count of events occurred?

File: docs/en/operations/system_tables.md [hunk 4]
```diff
@@ -89,9 +119,35 @@ Note that the amount of memory used by the dictionary is not proportional to the
...
+- `description` ([String](../data_types/string.md)) — Description of event.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: of **an** event

File: docs/en/operations/system_tables.md [hunk 5]
```diff
@@ -89,9 +119,35 @@ Note that the amount of memory used by the dictionary is not proportional to the
...
 ## system.events {#system_tables-events}
 
-Contains information about the number of events that have occurred in the system. This is used for profiling and monitoring purposes.
-Example: The number of processed SELECT queries.
-Columns: 'event String' – the event name, and 'value UInt64' – the quantity.
+Contains information about the number of events that have occurred in the system. For example, in the table, you can find how many `SELECT` queries are processed from the moment of ClickHouse server start.
+
+Columns:
+
+- `event` ([String](../data_types/string.md)) — Event name.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: Name of event **kind**?

File: docs/en/operations/system_tables.md [hunk 6]
```diff
@@ -133,14 +189,48 @@ Columns:
...
-- `total_size_marks UInt64` — The total number of marks in the merged partss.
+- `total_size_marks UInt64` — The total number of marks in the merged parts.
 - `bytes_read_uncompressed UInt64` — Number of bytes read, uncompressed.
 - `rows_read UInt64` — Number of rows read.
 - `bytes_written_uncompressed UInt64` — Number of bytes written, uncompressed.
 - `rows_written UInt64` — Number of lines rows written.
 
 ## system.metrics {#system_tables-metrics}
 
+Contains metrics which can be calculated instantly, or have an instant value. For example, a number of simultaneously processed queries, the current value for replica delay. This table is always up to date.
```
Comment (blinkov) [inline docs/en/operations/system_tables.md]: instant value -> current value

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/operations/system_tables.md (hunk 5 above)
Comment (BayoNet) [reply docs/en/operations/system_tables.md]: Is there a difference?

Kind as a word can accumulate differently named events in a group. Here we have a textual identifier of event. We can call it name or id, but I think kind is not right word.

**Comment by BayoNet** [comment]
Тут вроде всё хорошо. Вмёржить бы :-).

**Pull request merged**